app.js

```diff
@@ -1,23 +1,46 @@
 // Task 1 - Notes for JS Evaluation Predictions
 
 // List 3 expressions that surprised you or gave unexpected results, why?
+// 1. 2 + 'pizza' -> I was surprised to see when combining a number and a string, the result returned a combined string ('2pizza'). Similarly, I was surprised to see that when mulitplying a number by a string, it returned NaN.
+// 2. 5 === 5.0 -> It makes sense now that I've seen it, but I was surprised because I thought === only was truthy if the values were identical, I thought the .0 (although not adding any mathematical value), may have caused it to return false
+// 3. 'new york'[1] -> I had never seen an instance before where you determine the index of a string - but after testing it makes sense that the result is the letter which corresponds to the index value.
 
 // List 2 exprsssions you felt comfortable interpreting at a glance.
+// 1. 21 % 2 === 1 -> I am comfortable with % expressions, which determine the remainder, and using comparison operators to determine if the result is truthy or falsey when compared to a number
+// 2. !true && !(false || false) -> After reviewing the Js control flow lesson, I now feel more comfortable with operators and comparing two values for their truthy or falsiness. In this case it's using an && operator, and since the first value is not true (false), it stops immediately and returns false.
 
 // List 1 expression you are still 'shaky' on, why?
-
-
-
+// 1. 'engineering'.indexOf('G') -> I expected this result to be undefined, because the exact string 'G' does not exist, but it returned -1. I am under the impression when it comes to index values, -1 is the equivalent of undefined?
```

Comment:
Yeah you can think of `-1` as undefined. It just means whatever you're looking for does not exist